Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TurkishAirLinesHelper.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/19/17.
+//  Copyright © 2017 Modeso. All rights reserved.
+//
+
+import Foundation
+
+class TurkishAirLinesHelper {
+    
+    enum GameDataStatus {
```

Comment:
i think this enum name should be: requestype or GameDataType 
as it's not a status